src/plugins/intel_gpu/src/graph/program_helpers.cpp

```diff
@@ -202,7 +203,54 @@ bool program_helpers::needs_onednn_sum_post_op(const eltwise_node& n, layout inp
     return false;
 }
 
+bool onednn_add_fusing_helpers::is_full_tensor(const layout& l) {
+    if (l.size.spatial[0] > 1 || l.size.spatial[1] > 1 || l.size.batch[0] > 1) {
```

Comment:
I think we need to check spatial[2] as well for `zyx` format

Reply:
```suggestion
    if (l.size.spatial[0] > 1 || l.size.spatial[1] > 1 || l.size.batch[0] > 1) {
```
    if (l.size.spatial[0] > 1 || l.size.spatial[1] > 1 || (l.get_spatial_rank() == 3 && l.size.spatial[2] > 1)
        || l.size.batch[0] > 1) {